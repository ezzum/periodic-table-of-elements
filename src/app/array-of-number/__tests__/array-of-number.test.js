import React from "react";
import { mount } from "enzyme";

import ArrayOfNumber from "../array-of-number";

describe("array-of-number tests", () => {
  const props = {
    numberCount: 8,
  };

  const FakeComponent = (props) => {
    return <div>{props.numbers}</div>;
  };

  const Component = ArrayOfNumber(FakeComponent, "test");
  let wrapper;

  beforeEach(() => {
    wrapper = mount(<Component {...props} />);
  });

  it("snapshot test", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
