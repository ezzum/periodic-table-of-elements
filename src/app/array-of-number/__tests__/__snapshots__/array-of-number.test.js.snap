// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`array-of-number tests snapshot test 1`] = `ReactWrapper {}`;
